chat-plugins/info.js

```diff
@@ -328,7 +328,7 @@ var commands = exports.commands = {
 	dt: 'details',
 	details: function (target) {
 		if (!target) return this.parse('/help details');
-		CommandParser.commands.data.apply(this, arguments);
+		this.run('data');
```

Comment:
``this.run(commands.data.name)`` doesn't fix it due to https://code.google.com/p/v8/issues/detail?id=2276&q=function%20name&colspec=ID%20Type%20Status%20Priority%20Owner%20Summary%20HW%20OS%20Area%20Stars :[